sensu-plugins-pagerduty.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsPagerduty::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', '~> 2.0.0'
```

Comment:
this is over specific we follow semver so pinning to `~> 2.0` should be safe. We should pin to `~> 2.5` while we are at it since this is the most recent version released.